src/services/admin-api.ts

```diff
@@ -32,6 +46,18 @@ export default class AdminApi {
     this.routeBroadcaster = deps(RouteBroadcaster)
     this.rateBackend = deps(RateBackend)
     this.stats = deps(Stats)
+
+    this.routes = [
```

Comment:
With this many endpoints do you think it could be worth upgrading to something like koa?